Makefile

```diff
@@ -45,6 +45,8 @@ install:
 
 images:
 	@echo -e "\033[2mBuilding container $(CONTAINER_NAME_ARCH)\033[0m"
+	mkdir -p $(BIN_DIR)/$(ARCH)/assets/profiles
```

Comment:
@vishnuchalla Not related to this PR - but `ORG ?= vchalla` be updated to `openshift-lightspeed`